docs/generated.d.ts

```diff
@@ -3335,6 +3350,57 @@ export interface RpcAddressTxSubscriptionRequest {
   params: RpcAddressTxSubscriptionParams;
   [k: string]: unknown | undefined;
 }
+export interface RpcBlockNotificationResponse {
+  jsonrpc: "2.0";
+  method: "block";
+  params: Block;
+  [k: string]: unknown | undefined;
```

Comment:
additionalProperties: false Gets rid of the any key type, if it's not needed

Reply:
Great idea, done, thanks